src/components/Steps/SendTransaction/SubmitTransaction.vue

```diff
@@ -120,7 +120,7 @@ const isSuccess = computed(() => props.state.status === 'success');
 const { copy: onCopyAddress } = useClipboard();
 const txLink = computed(
   () =>
-    `https://explorer.aptoslabs.com/txn/${txState.value.hash}?network=mainnet `,
+    `https://explorer.aptoslabs.com/txn/${txState.value.hash}?network=mainnet`,
```

Comment:
network name should correspond to selected network: mainnet/testnet

Reply:
fixed